### DISCUSSION TITLE
Can I highlight filename with diagnostic colors?

### DISCUSSION THREAD

**Discussion by Martinits** [nvim-tree/nvim-tree.lua#1913] [Q&A]
Hello, I'd like to highlight filename with the diagnostic colors instead of showing diagnostic icons, but I didn't find out any config item under "diagnostic" ?

**Comment by alex-courtis** [comment]
We have highlight groups for the signs but no file equivalents.

#1917

**Reply by Martinits** [reply]
Thanks for reply! But I think it would be nice to have this configuration. We can configure git colors for both filename and signs, why not diagnostics?

**Reply by alex-courtis** [reply]
See the feature request above ;)
